src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-checkoutCredentialsId.html

```diff
@@ -1,3 +1,13 @@
 <div>
-    <p>Credentials used to check out sources during a build.</p>
+    <p>
+        Credentials used to check out sources during a build.
+    </p><p>
+        The default choice of "Same as scan credentials" is sufficient so long as you don't
+        need Jenkins to push some changes back into these repositories.
+    </p><p>
+        Otherwise you should create an "SSH username with private key" credential and
```

Comment:
I'll do as you say here, but I don't think such convention is established as a practice. FWIW, personally I found those bold letters distracting as it draws eyes too much.